cohere_vector/challenges/default.json

```diff
@@ -44,16 +44,38 @@
       }
     },
     {
-      "name": "knn-search-10-100",
+      "name": "standalone-search-knn-10-100",
       "operation": "knn-search-10-100",
       "warmup-iterations": 100,
-      "iterations": 1000
+      "clients": {{ standalone_search_clients | default(1) | int }},
+      "iterations": {{ standalone_search_iterations | default(10000) | int }}
     },
     {
-      "name": "knn-search-100-1000",
+      "name": "standalone-search-knn-100-1000",
       "operation": "knn-search-100-1000",
       "warmup-iterations": 100,
-      "iterations": 1000
+      "clients": {{ standalone_search_clients | default(1) | int }},
+      "iterations": {{ standalone_search_iterations | default(10000) | int }}
+    },
+    {
+      "parallel": {
+        "tasks": [
+          {
+            "name": "parallel-documents-indexing-bulk",
+            "operation": "parallel-documents-indexing",
+            "clients": {{ parallel_indexing_bulk_clients | default(1) | int }},
+            "time-period": {{ parallel_indexing_time_period | default(1800) | int }},
+            "target-throughput": {{ parallel_indexing_bulk_target_throughput | default(1) | int }}
```

Comment:
I guess we also need the bulk size to derive the throughput in terms of docs per second?

Reply:
This is right.